docs/content/dotnet-core.md

```diff
@@ -86,20 +86,21 @@ notes](https://github.com/paketo-buildpacks/dotnet-core-aspnet/releases).
 application declares its Runtime Framework as either `Microsoft.AspNetCore.App`
 or `Microsoft.AspNetCore.All`.
 
-### Using buildpack.yml
-
-To configure the buildpack to use .Net Core Runtime and ASP.Net v2.1.14 when
-deploying your app, include the values below in your `buildpack.yml` file:
+### Using BP_DOTNET_FRAMEWORK_VERSION
 
+To configure the buildpack to use .Net Core Runtime and ASP.Net v5.0.4 when
+deploying your app, set the following environment variable at build time,
+either directly (e.g. `pack build my-app --env
+BP_DOTNET_FRAMEWORK_VERSION=5.0.4`) or through a
+[project.toml](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md)
+file:
 {{< code/copyable >}}
----
-dotnet-framework:
-  version: "2.1.14"
+BP_DOTNET_FRAMEWORK_VERSION=5.0.4
```

Comment:
```suggestion
[build]
[[build.env]]
    name='BP_DOTNET_FRAMEWORK_VERSION'
    value='5.0.4'
```

Let's specify how you'd actually put this in a `project.toml`.

Reply:
I think for this part, we were just trying to show what the environment variable looks like. I don't think we need to show the project.toml case here, especialyl since we link out to how to use a project.toml. We don't do this anywhere else either.